### PR TITLE
3.4.1-1.1rc doc updates and release notes

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: source/changed_in_34.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@
 Changed in Percona Server for MongoDB 3.4
 =========================================
 
+The :ref:`mongorocks` storage engine is now based on RocksDB 4.11.2.
```
Comment (denis-protivenskii) [inline source/changed_in_34.rst]: Rocks 4.13.5

**Review by denis-protivenskii — CHANGES_REQUESTED** [review]

File: source/release_notes/3.4.1-1.1rc.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.. _3.4.1-1.1rc:
+
+=====================================
+ |PSMDB| 3.4.1-1.1 Release Candidate
+=====================================
+
+:Date: January 30, 2016
```
Comment (hrvojem) [inline source/release_notes/3.4.1-1.1rc.rst]: 31st, 2017

**Review by hrvojem — CHANGES_REQUESTED** [review]
* Bump the version in conf.py
* s/2016/2017 for copyright there as well

File: source/release_notes/3.4.1-1.1rc.rst [hunk 3]
```diff
@@ -4,7 +4,7 @@
  |PSMDB| 3.4.1-1.1 Release Candidate
 =====================================
 
-:Date: January 30, 2016
+:Date: January 31, 2016
```
Comment (hrvojem) [inline source/release_notes/3.4.1-1.1rc.rst]: 2017

**Pull request merged**